examples/reference/panes/HoloViews.ipynb

```diff
@@ -119,18 +207,205 @@
+   "metadata": {},
+   "source": [
+    "### Dynamic\n",
+    "\n",
+    "You can change the plotting backend dynamically via a widget too."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\", height=300, responsive=True, title=\"Try changing the backend\")\n",
+    "plot_pane = pn.pane.HoloViews(plot, backend='bokeh', sizing_mode=\"stretch_width\", height=300)\n",
+    "widget = pn.widgets.RadioButtonGroup.from_param(plot_pane.param.backend, button_type=\"primary\", button_style=\"outline\")\n",
+    "pn.Column(widget, plot_pane)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Linked Axes\n",
+    "\n",
+    "By default the axes of plots with shared key or value dimensions are linked. You can remove the link by setting the `linked_axes` parameter to `False`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "not_linked_plot = df.hvplot.scatter(x=\"group\", y=\"value\", responsive=True, title=\"Not Linked Axes\").opts(active_tools=['box_zoom'])\n",
+    "linked_plot = df.hvplot.scatter(x=\"group\", y=\"value\", responsive=True, title=\"Linked Axes\").opts(active_tools=['box_zoom'])\n",
+    "\n",
+    "\n",
+    "pn.Column(\n",
+    "    pn.pane.HoloViews(not_linked_plot, backend='bokeh', sizing_mode=\"stretch_width\", height=200, linked_axes=False),\n",
+    "    pn.pane.HoloViews(linked_plot, backend='bokeh', sizing_mode=\"stretch_width\", height=200),\n",
+    "    pn.pane.HoloViews(linked_plot, backend='bokeh', sizing_mode=\"stretch_width\", height=200),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Theme\n",
+    "\n",
+    "You can change the `theme`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\", height=300, responsive=True)\n",
+    "pn.pane.HoloViews(plot, height=300, theme=\"night_sky\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Layout and Widget Parameters\n",
+    "\n",
+    "Please note that above we used the `Holoviews` pane to display HoloViews objects.\n",
+    "\n",
+    "Below we will be using `pn.panel(...)` instead. `pn.panel(...)` is the same as `pn.pane.HoloViews(...).layout`\n",
+    "\n",
+    "### Center\n",
+    "\n",
+    "You can center your plots via the `center` parameter."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\", height=300, width=400)\n",
     "\n",
     "pn.Column(\n",
-    "    pn.Row(*widgets),\n",
-    "    hv_panel[0])"
+    "    \"## Not Centered: `center=False`\",\n",
```

Comment:
Does `center` work as expected when combined with `sizing_mode`? I don't think so. But I can find a way to achieve what I need.

![image](https://github.com/holoviz/panel/assets/42288570/06802c57-0808-4c91-8df0-c1aed4286c04)